models/validators.go

```diff
@@ -0,0 +1,8 @@
+package models
```

Comment:
I'd rather move this into an `input`/`api` validator rather than a `models` validator. I'd even put it into a `parser` module and return the actual `Time` rather than using a string within the model.